_docs/dev-setup.md

```diff
@@ -28,13 +29,53 @@ Oh wow, this looks like a lot! But trust us, this only takes 15 minutes, max.
   * [Make your Changes](#make-your-changes)
   * [Add, Commit, and Push](#add-commit-and-push)
 * [Next Steps](#next-steps)
-* [Appendix: GitHub Desktop](#appendix-github-desktop)
-* [Appendix: Installing Ruby with RVM](#appendix-installing-ruby-with-rvm)
+* [Appendix](#appendix)
```

Comment:
There is no appendix header, so you should add one I think!

Reply:
There is, but it has no content beneath it - just used to mark the section.